src/eko/msbar_masses.py

```diff
@@ -2,13 +2,14 @@
 r"""
 This module contains the |RGE| for the |MSbar| masses
 """
+import numba as nb
 import numpy as np
 from scipy import integrate, optimize
 
 from .beta import b, beta
 from .evolution_operator.flavors import quark_names
 from .gamma import gamma
-from .strong_coupling import StrongCoupling, strong_coupling_mod_ev
+from .strong_coupling import StrongCoupling, invert_matching_coeffs
 
 
 def msbar_ker_exact(a0, a1, order, nf):
```

Comment:
Here and below: as already mentioned, you're in the `msbar_masses` module, so everything that is `msbar` and `masses` you can omit, is the default.
If you compute/evolve/do something on something else, that instead you should specify.